api/helpers/monitor.js

```diff
@@ -44,17 +52,18 @@ module.exports = app => {
   function getStreetSide(date) {
     var day = date.getDate();
     if (day % 2 === 0) {
-      ``;
       return "Even";
     } else {
       return "Odd";
     }
   }
 
   //adds 72 hours to the starting date
+  //needed to add extra hour to be a true 72 hours from start date
   function getExpirationDate(date) {
```

Comment:
Try to use ES6 syntax

```suggestion
  getExpirationDate = date => {
```

Reply:
Okay, I'll update that when I get a chance